README.md

```diff
@@ -1,13 +1,34 @@
-# OurBooks
+# Books client
 
-## Build Setup
+### Configuration
 
+The project needs to be configured with your Auth0 domain and client ID in order for the authentication flow to work.
+
+To do this, first copy `auth_config.json.example` into a new file in the same folder called `auth_config.json`, and replace the values within with your own Auth0 application credentials:
+
+```json
+{
+  "domain": "<YOUR AUTH0 DOMAIN>",
+  "clientId": "<YOUR AUTH0 CLIENT ID>"
+}
+```
```

Comment:
I think we don't need that. Just hardcode credentials in javascript file.
Rationale: These are not the secrets so we don't have to care about security